addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -440,25 +443,27 @@
 					</control>
 -					<control type="label">
 						<left>50</left>
-						<top>0</top>
+						<top>5</top>
 						<right>80</right>
-						<height>60</height>
-						<font>font14</font>
+						<height>30</height>
 						<aligny>center</aligny>
-						<label>$INFO[ListItem.Label]$VAR[VideoStreamDialogAudioItemLabelVar, - [LIGHT],[/LIGHT]]</label>
+						<scroll>true</scroll>
+						<label>$INFO[ListItem.Label]$VAR[VideoStreamDialogItemLabelVar, - [LIGHT],[/LIGHT]]</label>
 					</control>
-					<control type="textbox">
+					<control type="label">
 						<left>50</left>
-						<top>50</top>
+						<top>45</top>
 						<right>80</right>
-						<height>80</height>
-						<font>font12</font>
+						<height>20</height>
+						<font>font10</font>
+						<aligny>center</aligny>
+						<scroll>true</scroll>
+						<label>$INFO[ListItem.Property(stream.codecdesc),, - ]$INFO[ListItem.Property(stream.channels), $LOCALIZE[31612] ]$INFO[ListItem.Property(stream.description), - ]</label>
```

Comment:
there is a surplus space
```suggestion
<label>$INFO[ListItem.Property(stream.codecdesc),, - ]$INFO[ListItem.Property(stream.channels),$LOCALIZE[31612] ]$INFO[ListItem.Property(stream.description), - ]</label>
```

Reply:
I will check, but I'm sure the space is needed before the Channels label to separate from  codedesc value. Do we even the Channels value? as channel layout always seems to be part of codedesc, unless you see cases where it's not present from addon streams.

Reply:
Actually I probably do need to some adjustments for this since as I was trying to account for possibility of no codedesc value. However as CrystalP has found cause of no codedesc value for DVD image/folder files, then there's no need to account for this anymore.

Reply:
stream.codecdesc have suffix of " - " a space is already included should in theory be enough

stream.codecdesc could include a "channel" string value, but it come from ffmpeg metadata such as mkv,
with all others demuxers such as pvr/inpustreams etc... dont exists

I would have thought of removing this
https://github.com/xbmc/xbmc/blob/25915bd45c9f4f4cf97dddeb1c2b88923a47ffaa/xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.cpp#L129
so that the codec description stays clean

it would probably be more appropriate to have our own text conversion of the channels value instead of outsourcing the task to ffmpeg, so that can be used by all demuxers

Reply:
Adjustments made